Add the troubleshooting section

luleonpla opened the pull request:
Error AADSTS50011: The reply address ‘https://contoso.com’ does not match the reply addresses configured for the application*

azurecla commented:
Hi __@damienpontifex__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

davidwrede commented:
Hi damienpontifex,

Thanks again for updating our documentation with a new screenshot and steps! It's very much appreciated.

- Dave

luleonpla committed:
Add the troubleshooting section

Error AADSTS50011: The reply address ‘https://contoso.com’ does not match the reply addresses configured for the application*

NiyaNagi reviewed (approved):
double sweet!
